server/queues/processors/NotificationsProcessor.ts

```diff
@@ -99,12 +100,31 @@ export default class NotificationsProcessor extends BaseProcessor {
         );
       }
     }
+
+    // send notifs to mentioned users
+    const mentions = parseMentions(document);
```

Comment:
I think we should send the mentioned emails _first_ and then send to the document notification recipients. If anyone was mentioned in the document we can suppress the other email so they don't receive both…

Reply:
And we don't want to suppress _mention_ notifications at all?

Reply:
Currently both notifications should be sent – I think we'll tackle grouping and delay of notifications separately at that level of the system once #4924 is complete.